recipes-connectivity/network-manager-applet/network-manager-applet-1.4.4/com.openxt.xml

```diff
@@ -0,0 +1,10 @@
+<node>
+    <interface name="com.openxt.nmapplet">
+        <method name="popup_network_menu">
+            <arg name="x_ordinate" type="u" direction="in" />
```

Comment:
IMHO x_ordinate/y_ordinate are a bit unorthodox, unless your a mathematician. Did you consider x_coord/y_coord which I personally seen more often as the short hand for x/y coordinate variables.

Reply:
Ack, will change to x_coord/y_coord. Thanks